connector/tests/src/test/java/org/oaebudt/edc/ManagementApiTransferTest.java

```diff
@@ -287,4 +344,55 @@ public JsonObject createCallback(final String url, final boolean transactional,
                 .build();
     }
 
+    private static String createParticipantManifest(int protocolPort, int credentialsPort, String participantId, String participantName)
+    {
+        String base64ParticipantId = encodeBase64(participantId);
+
+        return String.format("{\n" +
+                "    \"roles\": [],\n" +
+                "    \"serviceEndpoints\": [\n" +
+                "        {\n" +
+                "            \"type\": \"CredentialService\",\n" +
+                "            \"serviceEndpoint\": \"http://localhost:%d/api/credentials/v1/participants/%s\",\n" +
+                "            \"id\": \"%s-credentialservice-1\"\n" +
+                "        },\n" +
+                "        {\n" +
+                "            \"type\": \"ProtocolEndpoint\",\n" +
+                "            \"serviceEndpoint\": \"http://localhost:%d/protocol\",\n" +
+                "            \"id\": \"%s-dsp\"\n" +
+                "        }\n" +
+                "    ],\n" +
+                "    \"active\": true,\n" +
+                "    \"participantId\": \"%s\",\n" +
+                "    \"did\": \"%s\",\n" +
+                "    \"key\": {\n" +
+                "        \"keyId\": \"%s#key-1\",\n" +
+                "        \"privateKeyAlias\": \"%s#key-1\",\n" +
+                "        \"keyGeneratorParams\": {\n" +
+                "            \"algorithm\": \"EC\"\n" +
+                "        }\n" +
+                "    }\n" +
+                "}", credentialsPort, base64ParticipantId, participantName, protocolPort, participantName, participantId, participantId, participantId, participantId);
```

Comment:
nit: this is really hard to read/maintain, maybe building the json using the `Json.createObjectBuilder` method will make everything easier.